packages/app/style.css

```diff
@@ -0,0 +1,5 @@
+@tailwind base;
```

Comment:
I think this file is not needed.

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317